js/form.js

```diff
@@ -0,0 +1,66 @@
+import {picturesContainer} from './renderingPicture.js';
+
+const VALID_SYMBOLS = /^#[a-zа-яё0-9]{1,19}$/i;
+const imgUploadForm = document.querySelector('.img-upload__form');
+const fileFild = picturesContainer.querySelector('.img-upload__input');
+const filtersForm = picturesContainer.querySelector('.img-upload__overlay');
+const battonHideFiltorsForm = filtersForm.querySelector('.img-upload__cancel');
+const inputTag = filtersForm.querySelector('.text__hashtags');
+const inputComment = filtersForm.querySelector('.text__description');
+
+const pristine = new Pristine(imgUploadForm,{
+  classTo: '.img-upload__field-wrapper',
```

Comment:
Точки не нужны по докам в названиях классов
```suggestion
  classTo: 'img-upload__field-wrapper',  
  errorTextParent: 'img-upload__field-wrapper',
```